lite2/client.go

```diff
@@ -208,22 +333,25 @@ func (c *Client) TrustedHeader(height int64, now time.Time) (*types.SignedHeader
 	if err != nil {
 		return nil, err
 	}
+	if h == nil {
```

Comment:
When would this happen - should we return an error?

Reply:
Someone requests outdated header (outside of trusting period - removed) OR header which is not yet verified (saved to `trustedStore`)? There's no error per se - light client just does not have the header.

Reply:
Hmm OK, as long as the caller handles `nil` correctly in all cases.